sway/desktop/output.c

```diff
@@ -228,11 +228,19 @@ static void render_view(struct sway_view *view, struct sway_output *output) {
  */
 static void render_container_simple_border_normal(struct sway_output *output,
 		struct sway_container *con, struct border_colors *colors,
-		struct wlr_texture *title_texture) {
+		struct wlr_texture *title_texture, int iteration) {
```

Comment:
I'd name this `depth` instead of `iteration`, but is just a personal preference.